src/logic/safe/store/actions/utils.ts

```diff
@@ -72,13 +73,20 @@ export const extractRemoteSafeInfo = async (remoteSafeInfo: SafeInfo): Promise<P
   const safeInfo: Partial<SafeRecordProps> = {
     modules: undefined,
     spendingLimits: undefined,
+    spendingLimitEnabled: false,
   }
   const safeInfoModules = (remoteSafeInfo.modules || []).map(({ value }) => value)
 
   if (safeInfoModules.length) {
     safeInfo.modules = buildModulesLinkedList(safeInfoModules)
+    safeInfo.spendingLimitEnabled =
+      safeInfoModules?.some((module) => sameAddress(module, SPENDING_LIMIT_MODULE_ADDRESS)) ?? false
```

Comment:
You don't need optional chaining here because you already check the length property on line 80.